### DISCUSSION TITLE
TheAgencyTheme (or any theme) change CSS

### DISCUSSION THREAD

**Discussion by LMSSonos** [OrchardCMS/OrchardCore#8511] [Q&A]
Let's assume I want to use `TheAgencyTheme` for my website. Next to the content I want to change some colors and background images. Is that something I can achieve via the /admin or do I have to take the Theme and Recompile it to my own version of it?

I know in the templates I can modify the HTML but I did not find a way to change css.

Thanks

**Comment by Skrypt** [comment]
You can create your own Layout admin template and modify the .css file used by your current theme by adding your custom .css file. That's one way to get around the need to recompile the current theme.

**Reply by LMSSonos** [reply]
Thanks for the hint. I modified the `Content__LandingPage` and added following snipped at the top

```
{% zone "HeadMeta" %}
<link href='/css/bar.css' rel='stylesheet' type='text/css'>
{% endzone %}
```

Result is as expected. Was this as well your suggestions? Thanks

**Reply by Skrypt** [reply]
Depending on the recipe/templates used this works too.

**Reply by LMSSonos** [reply]
Sorry that I'm asking, but from your answer I get you were thinking of another way. What do you mean by "Layout admin template" -> create a new Project the same way as building a template?

**Reply by Skrypt** [reply]
You can enable the "Template" feature to be able to create Liquid templates from the Admin.
So basically for example you could copy the layout.liquid file from TheBlogTheme to be served from the database.
All you need to do is create a liquid template who's using the "Layout" name.

![orchard](https://user-images.githubusercontent.com/3228637/107080359-2b6e5e80-67bf-11eb-9447-4957e55b63f2.gif)

**Reply by Skrypt** [reply]
Same thing for TheAgencyTheme. Works as fine.

**Reply by LMSSonos** [reply]
That's awesome. Thank you very much!! really appreciate the time you took to make the recording.

It never came to my mind that I can create the main template here as well. This is really cool. 

But one question that remains open for me is how do you manage the CSS? This one can't be modified through the admin? My quick fix was to add it to the root

**Reply by Skrypt** [reply]
You can use the media library to upload a .css file and then link it in your template.
Of course, we would need to have something that is better here to allow to edit the current theme .SCSS file directly from the admin UI eventually "and compile it". But, for now, this is the workaround that works.

So, the .css file uploaded in the media gallery would require to be added after your actual theme .css and it would override it's current styles.

**Reply by LMSSonos** [reply]
Thanks for the workaround, I could live with that ;-) the .SCSS option would be great, but to be honest, once the .css is configured it's not really changed that often.

**Reply by thecodecat23** [reply]
Hi there! I hope you're doing well. I'm trying to get the layout.liquid file while executing the app, but I'm having trouble with it. Could you provide any guidance or resources that could help me understand how to access this file? Any help would be greatly appreciated. Thanks in advance!
